src/app/components/AssignmentDutySplitter.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import TimeSlider from './FormElements/TimeSlider';
+import {
+  SheriffDuty, 
+  TimeType
+} from '../api/Api';
+
+export interface DutySplitterProps {
+  sheriffDuties: SheriffDuty[];
+  dutyStartTime: TimeType;
+  dutyEndTime: TimeType;
+}
+
+export default class DutySplitter extends React.Component<DutySplitterProps> {
+  render() {
+    const {
+      sheriffDuties,
+      dutyStartTime,
+      dutyEndTime
+    } = this.props;
+
+    return (
+      <div>
+        {
+          sheriffDuties.map((sheriffDuty) =>
+            <div key="sheriffDuty" style={{paddingBottom: 50}}>
```

Comment:
*key* should be unique.  So we will likely want it to be the id of the sheriffDuty